contracts/src/v0.8/dev/KeeperRegistryDev.sol

```diff
@@ -686,4 +713,20 @@ contract KeeperRegistryDev is
     if (msg.sender != owner() && msg.sender != s_registrar) revert OnlyCallableByOwnerOrRegistrar();
     _;
   }
+
+  /**
+   * @dev Reverts if called on a paused upkeep.
+   */
+  modifier onlyNonPausedUpkeep(uint256 id) {
+    if (s_upkeep[id].paused) revert OnlyNonPausedUpkeep();
+    _;
+  }
+
+  /**
+   * @dev Reverts if called on a non paused upkeep.
+   */
+  modifier onlyPausedUpkeep(uint256 id) {
```

Comment:
in addition to the issue above, this modifier is only used once, which I think usually makes it unnecessary

Reply:
One thing it would be helpful to list out all the functionality we want to prevent when upkeep is paused. I think it would be more than just one function.

Reply:
Done